### ISSUE TITLE
Build framework Research areas, Research lines and Projects for CPG website

### ISSUE THREAD

**Issue by MeykeHermsen** [DIAGNijmegen/website-content#304]
We agreed with Geert, Francesco and Jeroen upon the following replacement for 'Projects' on the CPG page.

Four research areas:
1. AI-assisted pathology diagnosis
2. Data-driven biomarker discovery
3. Basic computational pathology research
4. Infrastructure and challenges

Under research areas we have research lines, for example AI-assisted pathology diagnosis; Gleason grading.
Under research lines come specific projects, for example AI-assisted pathology diagnosis; Gleason grading; Deep PCA.

The research areas will be displayed on the research page of DIAG.

Can you build the framework for this? I will plan a meeting to discuss which projects should go where.

**Comment by bramvanginneken** [comment]
Great to have the same groups on DIAG site, experience learns we never maintain things well, so the more the same the better. I would rephrase nr 2 to Pathology biomarker discovery, and 3 is not an exciting title. I’m not sure what will be under 4 but if we keep this title we should not only put pathology projects in there.

**Comment by wouterbulten** [comment]
Research areas will be implemented as "normal research" pages. Research pages will get the ability to include a list of sub-research pages. For each research page, we will also add a flag whether to show that page in the research overview.

For example:

- Research page: _AI-assisted pathology diagnosis_, `show_in_overview: true`
  -  Research page: _Prostate cancer_, `show_in_overview: false`
     - Project page: _Gleason grading_
     - Project page: _PANDA Challenge_
  - etc.

**Comment by MeykeHermsen** [comment]
Hi webteam,

Are there any updates on this issue? New PhD students are starting in the group and they can now not be 'connected' to a specific project. Did you have a meeting since my last email in october with Francesco/Geert/Jeroen to come up with names for the research areas/lines and projects or should we make a draft together? Let me know if I can help.

**Comment by martvanrijthoven** [comment]
Hi Meyke, 

Yes actually almost done. The idea is to have it running first on develop.diagnijmegen.nl, such that we can work on the research lines and see how it looks like. I did not have a meeting with Francesco/Geert/Jeroen.  I will send an email this week to ask if they could help to finish this issue by defining the research lines and if somebody could help me generate the content. I will add you to cc. First, i have to test it on develop.diagnijmegen.

**Comment by MeykeHermsen** [comment]
great! I have forwarded you the last communication about this topic with G/F/J.

**Comment by martvanrijthoven** [comment]
develop website has now the pathology website online for developing purposes

**Comment by nlessmann** [comment]
Is this done except for adding the content? Then we can close the issue, I think.

**Comment by keelinm** [comment]
This is low priority for now since Marina does not get feedback from the CPG team on which content to add.
